compiler/rustc_lint_defs/src/builtin.rs

```diff
@@ -8,6 +8,135 @@ use crate::{declare_lint, declare_lint_pass, FutureIncompatibilityReason};
 use rustc_span::edition::Edition;
 use rustc_span::symbol::sym;
 
+declare_lint_pass! {
```

Comment:
I moved this to the top, it was in a random spot in the middle of the file which surely doesn't make a ton of sense.